### ISSUE TITLE
Circular dependencies problem in archivematica-mcp-client for ubuntu 14.04

### ISSUE THREAD

**Issue by scollazo** [artefactual-labs/am-packbuild#111]
The package fails to install with
<pre>
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:
The following packages have unmet dependencies:
 archivematica-mcp-client : Depends: ffmpeg (>= 7:3.0) but it is not going to be installed
                            Depends: fits (>= 0.8.4) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
</pre>

But it works when fits is installed in advance. As Ubuntu 14.04 is in maintenance mode, we can fix this in the am-docs

**Comment by mamedin** [comment]
Trying to install fits in Ubuntu 14 shows the following error:

```
fits : Depends: default-jdk but it is not going to be installed
```

I tried installing `default-jdk`:

```
The following packages have unmet dependencies:
 default-jdk : Depends: default-jre (= 2:1.7-51) but it is not going to be installed
               Depends: openjdk-7-jdk (>= 7~u3-2.1.1) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

When trying to install `openjdk-7-jdk`:

```
The following packages have unmet dependencies:
 openjdk-7-jdk : Depends: openjdk-7-jre (= 7u151-2.6.11-2ubuntu0.14.04.1) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

Finally I could install the `openjdk-7-jre` package that added the `libgl1-mesa-glx-lts-utopic` package as a dependency and it fixes the `ffmpeg` issue too.


If the `openjdk-7-jre` package is installed previously, this circular dependency does not happen.

**Comment by scollazo** [comment]
There is a pull request waiting for CR here: https://github.com/artefactual/archivematica-docs/pull/80

**Comment by sevein** [comment]
@scollazo if you want artefactual/archivematica-docs#80 merged it I recommend to add it to the 1.7 project board and update the pull request with a new comment so @jrwdunham knows that you're done with it.

**Comment by scollazo** [comment]
Done. As @jrwdunham started the CR, I didn't knew about the project board process.
